admin/api/attempts.go

```diff
@@ -32,5 +33,9 @@ func (api *API) GetAttempt(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	attemptDetail, err := api.DB.AttemptDetailsWS.Get(r.Context(), attempt.ID)
```

Comment:
attempts that have not been delivered would have no detail. I think doing a condition check before querying database would be more efficient. (can be done in another separate PR).

Reply:
added attempt delivered check